plugins/woocommerce/tests/e2e-pw/tests/merchant/create-order.spec.js

```diff
@@ -299,10 +299,17 @@ test.describe(
 			await page
 				.getByRole( 'textbox', { name: 'Postcode' } )
 				.fill( '12345' );
-			await page
-				.getByRole( 'textbox', { name: 'Select an option…' } )
-				.click();
-			await page.getByRole( 'option', { name: 'Florida' } ).click();
+			// eslint-disable-next-line playwright/no-conditional-in-test
+			if (
+				await page
+					.getByRole( 'textbox', { name: 'Select an option…' } )
+					.isVisible()
```

Comment:
The test _assumes_ that the billing address by default is in the US, which is not the case on the Pressable website. I'm putting 'Select an option…' in `if` block. It should not affect tests against wp-env